assembly/runtime/api/others.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * The rest of runtime entries for the Polkadot Host
+ * These methods are mocked for this iteration and they return an empty u8 array by default
+ */
+import { BytesReader } from "as-scale-codec";
+import { Log, Serialiser, SignedTransaction } from "subsembly-core";
+import { AccountId } from 'subsembly-core';
+import { Executive, System } from "../../frame";
+
+/**
+ * 
+ * @param data i32 pointer to the start of the argument passed
+ * @param len i32 length (in bytes) of the arguments passed
+ */
+export function BabeApi_configuration(data: i32, len: i32): u64 {
+    return Serialiser.serialiseResult([]);
+}
+
+/**
+ * 
+ * @param data i32 pointer to the start of the argument passed
+ * @param len i32 length (in bytes) of the arguments passed
+ */
+export function SessionKeys_generate_session_keys(data: i32, len: i32): u64 {
+    return Serialiser.serialiseResult([]);
+}
+
+/**
+ * Receives encoded byte array of Extrinsic appended to the source of transaction
+ * Returns ValidTransaction or TransactionError code
+ * @param data i32 pointer to the start of the argument passed
+ * @param len i32 length (in bytes) of the arguments passed
+ * source: TransactionSource, utx: Block::Extrinsic
+ * len + source + ext
+ */
+export function TaggedTransactionQueue_validate_transaction(data: i32, len: i32): u64 {
+    let input = Serialiser.deserialiseInput(data, len);
+    Log.info("Incoming tx: " + input.toString());
+    const uxt = BytesReader.decodeInto<SignedTransaction>(input);
```

Comment:
I think that we should use SignedTransactionType or smth like that (user-defined)